cypress.e2e.config.ts

```diff
@@ -0,0 +1,100 @@
+import { defineConfig } from 'cypress'
+import webpackPreprocessor from '@cypress/webpack-preprocessor'
+import { PNG } from 'pngjs'
+import pixelmatch from 'pixelmatch'
+import fs from 'fs'
+import path from 'path'
+
+const downloadDirectory = path.join(__dirname, '..', 'downloads')
+
+const checkFileDownloaded = async (filename: string, timeout: number, delayMs = 10): Promise<string | undefined> => {
+    const start = Date.now()
+    const fullFileName = `${downloadDirectory}/${filename}`
+
+    while (Date.now() - start < timeout) {
+        await new Promise((res) => setTimeout(res, delayMs))
+
+        if (fs.existsSync(fullFileName)) {
+            return fullFileName
+        }
+    }
+}
+
+import { createEntry } from './webpack.config'
```

Comment:
Imports should always be at the top of the file. Not sure why this wasn't flagged or causing issues.